build.gradle

```diff
@@ -46,7 +46,14 @@ configurations {
 dependencies {
 	implementation 'org.springframework.boot:spring-boot-starter-actuator'
 	implementation 'org.springframework.boot:spring-boot-starter-data-mongodb'
-	implementation 'org.springframework.boot:spring-boot-starter-web'
+	implementation( 'org.springframework.boot:spring-boot-starter-web:2.3.2.RELEASE') {
```

Comment:
shouldn't you update gradle plugin: https://plugins.gradle.org/plugin/org.springframework.boot

Reply:
you're right, now the right version of tomcat is loaded implicitly